pit/tests/table_basic_query_test.go

```diff
@@ -0,0 +1,447 @@
+	results, resp, err := apiClient.ExecutionAPI.
+		GetQueryError(ctx, queryId).
+		Execute()
+
+	require.NoError(t, err)
+	require.Equal(t, http.StatusOK, resp.StatusCode)
+
+	return results.Problems, resp, err
+}
+
+func copyData(t *testing.T, apiClient *apiclient.APIClient, ctx context.Context, tableId string, filename string, hasHeader bool, columnOrder []string) (string, *http.Response, error) {
+	copyQuery := apiclient.NewCopyQuery(filename, tableId)
+	copyQuery.SetDoesCsvContainHeader(hasHeader)
+	copyQuery.SetDestinationColumns(columnOrder)
+
+	queryId, resp, err := apiClient.ExecutionAPI.SubmitQuery(ctx).
+		ExecuteQueryRequest(
+			*apiclient.NewExecuteQueryRequest(
+				apiclient.CopyQueryAsQueryQueryDefinition(copyQuery),
+			),
+		).
+		Execute()
+
+	require.NoError(t, err)
+	require.Equal(t, http.StatusOK, resp.StatusCode)
+
+	return queryId, resp, err
+}
+
+func selectData(t *testing.T, apiClient *apiclient.APIClient, ctx context.Context, tableId string, mayFail bool) (string, *http.Response, error) {
+	selectQuery := apiclient.NewSelectQuery()
+	selectQuery.SetTableName(tableId)
+
+	queryId, resp, err := apiClient.ExecutionAPI.SubmitQuery(ctx).
+		ExecuteQueryRequest(
+			*apiclient.NewExecuteQueryRequest(
+				apiclient.SelectQueryAsQueryQueryDefinition(selectQuery),
+			),
+		).
+		Execute()
+
+	if !mayFail {
+		require.NoError(t, err)
+		require.Equal(t, http.StatusOK, resp.StatusCode)
+	}
+
+	return queryId, resp, err
+}
+
+func TestBasicQueries(t *testing.T) {
+	dbClient := pit.DbClient(BaseURL)
+	ctx := context.Background()
+
+	// Read the people table schema from file
+	peopleSchema, err := readPeopleSchema()
+	require.NoError(t, err)
+
+	csvColumnOrder := []string{"id", "name", "surname", "age"}
+
+	filePathWithoutHeader, err := filepath.Abs(filepath.Join("..", "tables", "people", "data_without_headers.csv"))
+	require.NoError(t, err)
+
+	filePathWithHeader, err := filepath.Abs(filepath.Join("..", "tables", "people", "data.csv"))
+	require.NoError(t, err)
+
+	filePathLong, err := filepath.Abs(filepath.Join("..", "tables", "people", "data_long.csv"))
+	require.NoError(t, err)
+
+	filePathTooManyColumns, err := filepath.Abs(filepath.Join("..", "tables", "people", "data_too_many_columns.csv"))
+	require.NoError(t, err)
+
+	t.Run("TableCopyWithoutHeaderAndSelect", func(t *testing.T) {
+		tableId := createTableWithCleanup(t, dbClient, ctx, peopleSchema)
+
+		queryId, _, _ := copyData(t, dbClient, ctx, tableId, filePathWithoutHeader, false, csvColumnOrder)
+		_, _, _ = waitForQueryToComplete(t, dbClient, ctx, queryId)
+
+		queryId, _, _ = selectData(t, dbClient, ctx, tableId, false)
+		_, _, _ = waitForQueryToComplete(t, dbClient, ctx, queryId)
+
+		results, _, _ := getQueryResults(t, dbClient, ctx, queryId, nil, nil, false)
+		// TODO what's the point of results array?
```

Comment:
techincally, this should be sorted and comapred with CSV data. But it works only for small tables.